generator/templates/cli/retrieveflag.gotmpl

```diff
@@ -22,6 +22,17 @@
             return err, false
         }
         m.{{ pascalize .Name }} = {{- if .IsNullable }}&{{- end }}{{ $flagValueVar }}
+    {{- else if eq .GoType "strfmt.ObjectId" }} {{/*Get flag value as string, then parse it*/}}
+        {{ template "flagnamevar" . }}
+        {{ $flagValueVar }}Str, err := cmd.Flags().GetString({{ $flagNameVar }})
+        if err != nil{
+            return err, false
+        }
+        {{ $flagValueVar }} := strfmt.ObjectId{}
+		if err := {{ $flagValueVar }}.UnmarshalText([]byte({{ $flagValueVar }}Str)); err != nil{
+            return err, false
+        }
+        m.{{ pascalize .Name }} = {{- if .IsNullable }}&{{- end }}{{ $flagValueVar }}
     {{- else if eq .GoType "strfmt.UUID" }} {{/*Many of the strfmt string alias types can be added here*/}}
```

Comment:
Maybe 
```
{{- else if or (eq .GoType "strfmt.UUID") (eq .GoType "strfmt.ObjectId") }}
```
Can work without copy paste the majority of the texts

Reply:
They all use different form for initialization. For example you can not init UUID with {{ .GoType }}{} and ObjectId fails when initialized with empty string. However, if we use:

> var {{ $flagValueVar }} {{ .GoType }}

The same code works for all 3 cases. I will update the code soon.

Reply:
fixed as described.